must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,20 +191,36 @@ for ns in $namespaces; do
         done
     done
 
-    # Add Ready nodes to the list
+    #Add ocs ready nodes to the list
     nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
 
-    # Collecting ceph prepare volume logs
```

Comment:
add this comment above the volume collection func

Reply:
It is above the volume collection func